packages/components/src/lib/queries/lexical-query.js

```diff
@@ -274,19 +299,23 @@ export default class LexicalQuery extends Query {
         adapterLexiconResShort.errors.forEach(error => this.logger.log(error.message))
       }
 
-      const adapterLexiconResFull = yield ClientAdapters.lexicon.alpheios({
-        method: 'fetchFullDefs',
-        clientId: this.clientId,
-        params: {
-          opts: lexiconFullOpts,
-          homonym: this.homonym,
-          callBackEvtSuccess: LexicalQuery.evt.FULL_DEFS_READY,
-          callBackEvtFailed: LexicalQuery.evt.FULL_DEFS_NOT_FOUND
-        }
-      })
+      let adapterLexiconResFull = {}
+      // Do not retrieve full definition for wordlist requests
+      if (this._source !== LexicalQuery.sources.WORDLIST) {
+        adapterLexiconResFull = yield ClientAdapters.lexicon.alpheios({
```

Comment:
@kirlat , why did you decide to turn off full homonyms for wordlist?
We save them to IndexDB source and show when a user clicks on a word in wordlist.

Reply:
Oh, I believe I know, why. Because we use only short definitions. 
I didn't turn off it myself in my PR because I believe that it could be usefull to update data and for full homonyms too, as this process is async and won't make delays for quizlet download.

Reply:
So if we do not need full definitions for word list right now maybe we can disable its retrieval for now to make things faster, but we could enable it later. @balmas, @irina060981, what do you think?

Reply:
yes, I think we can disable it for now.

Reply:
By the way - it won't make things faster, as full definitions are retrieved completely async, and it doesn't influence on a delay before download starts

Reply:
not for the UI but it could add load to the server for unnecessary requests